Cargo.toml

```diff
@@ -5,3 +5,11 @@ edition = "2024"
 
 [dependencies]
 clap = { version = "4.5.53", features = ["derive"] }
+serde = { version = "1.0", features = ["derive"] }
+toml = "0.8"
+dialoguer = "0.11"
+dirs = "5.0"
+thiserror = "2.0"
+
+[dev-dependencies]
```

Comment:
`tempfile` is added as a dev-dependency per the implementation plan, but the current tests don't use it for temporary directory testing. Consider either removing the unused dependency or adding tests that use temp directories to test `load()`/`save()` with actual file I/O.